src/Sulu/Component/Util/Tests/Unit/SortUtilsTest.php

```diff
@@ -105,16 +105,26 @@ public function provideSortObjects()
                 '[foo]', ['2', '1', '3'],
             ],
 
-            // multi dimensional array missing key
+            // multi dimensional array missing key asc
```

Comment:
two null values are different, to test the order I created a asc and desc test based on object:

See also https://3v4l.org/LsUj7